### PR TITLE
test: set button text white

### PR DESCRIPTION
**Pull request by danielgent**
:exclamation: *DELETE THE DESTINATION BRANCH FIRST MARKUS* :exclamation: _THIS PR IS CHAINED_ :chains: :chains: :chains: :chains: :chains: :chains:

### CODE REVIEW

**Comment by danielgent** [comment]
:exclamation: *DELETE THE DESTINATION BRANCH FIRST MARKUS* :exclamation: _THIS PR IS CHAINED_ :chains: :chains: :chains: :chains: :chains: :chains:

File: test/integration/flows/__image_snapshots__/password-spec-ts-test-integration-flows-password-spec-ts-password-flow-password-reset-should-reject-non-matching-confirmation-password-1-snap.png
Comment (danielgent) [inline test/integration/flows/__image_snapshots__/password-spec-ts-test-integration-flows-password-spec-ts-password-flow-password-reset-should-reject-non-matching-confirmation-password-1-snap.png:1]: Cool how the snapshots just update! 


I think it makes sense to alternate the vendor settings per test so we test the styling as well 


I could create different mocks for `fetchVendorSettings` *OR* the existing mock could respond to the tenant name... but then we'd need more fixtures _and_ there'd be logic

Maybe `getEnv` could accept an optional vendor settings object

File: test/integration/helpers/test-client.ts [hunk 1]
```diff
@@ -213,7 +213,7 @@ export async function getEnv() {
         logoUrl: `https://assets.sesamy.com/static/images/email/sesamy-logo.png`,
         style: {
           primaryColor: "#7D68F4",
-          buttonTextColor: "#7D68F4",
+          buttonTextColor: "#FFFFFF",
```
Comment (danielgent) [inline test/integration/helpers/test-client.ts:216]: I expected more to change *BUT* the emails are styled based on auth2's tenant info, where as the login pages are styled from the vendor settings of course :smile:

File: test/integration/flows/__image_snapshots__/password-spec-ts-test-integration-flows-password-spec-ts-password-flow-password-reset-should-reject-non-matching-confirmation-password-1-snap.png
Comment (markusahlstrand) [reply test/integration/flows/__image_snapshots__/password-spec-ts-test-integration-flows-password-spec-ts-password-flow-password-reset-should-reject-non-matching-confirmation-password-1-snap.png:1]: I would really like to move the vendor settings into auth2 so we don't have external dependencies.. But I'll finish up the openapi migration before starting up new projects..

**Review by markusahlstrand — APPROVED** [review]
Cool with the "onion skin" mode on the comparison :)

**Pull request merged**